plugins/out_cloudwatch_logs/cloudwatch_logs.c

```diff
@@ -148,6 +148,16 @@ static int cb_cloudwatch_init(struct flb_output_instance *ins,
         ctx->role_arn = tmp;
     }
 
+    tmp = flb_output_get_property("sts_endpoint", ins);
+    if (tmp) {
+        ctx->sts_endpoint = (char *) tmp;
+    }
+
+    if (ctx->sts_endpoint && !ctx->role_arn) {
+        flb_plg_error(ctx->ins, " 'aws_role_arn' is a required option to use "
```

Comment:
In the CloudWatch plugin its just called `role_arn`

Reply:
Also see my other comment, I just realized this error isn't true or needed since we use sts_endpoint in the EKS provider